package.json

```
@@ -11,6 +11,7 @@
         "format": "prettier --write .",
         "lint": "prettier --check . && eslint .",
         "test": "npm run test:unit && npm run test:integration && npm run test:e2e",
+        "show-coverage": "npm run test && cd coverage/ && open index.html",
```

How about:
```suggestion
        "show-coverage": "open coverage/index.html",
```
I often run only the integration test and then want to open the coverage, so I think we should have that separated. We could also add a command that combines both, e.g. `test:show-coverage`

---

I think your second suggestion sounds like a good idea, I will add this command and adapt the first one.